css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
+	user-select: none;
+}
+
+.spectateBanner::before{
+	content: "👁️ ";
+	font-style: normal;
+}
 .spectateBanner{
 	background-color: #90FF90;
 	cursor: pointer;
-	text-align:center;
+	flex: 0 0 35%;
+	width: 99.5%; /* I think this is due to a padding issue or something? */
+	text-align: center;
+	font-style: italic;
+	background-color: lightgreen;
```

Comment:
you already have a background-color set to 90FF90. replace/remove that line.

Reply:
I opted to remove 90FF90 and keep the lightgreen instead since it looks better, but either way fixed.